astropy/coordinates/angles.py

```diff
@@ -33,24 +33,24 @@ class Angle(u.SpecificTypeQuantity):
 
     The input parser is flexible and supports a variety of formats::
 
-      Angle('10.2345d')
-      Angle(['10.2345d', '-20d'])
-      Angle('1:2:30.43 degrees')
-      Angle('1 2 0 hours')
-      Angle(np.arange(1, 8), unit=u.deg)
-      Angle('1°2′3″')
-      Angle('1°2′3″N')
-      Angle('1d2m3.4s')
-      Angle('1d2m3.4sS')
-      Angle('-1h2m3s')
-      Angle('-1h2m3sE')
-      Angle('-1h2.5m')
-      Angle('-1h2.5mW')
-      Angle('-1:2.5', unit=u.deg)
-      Angle((10, 11, 12), unit='hourangle')  # (h, m, s)
-      Angle((-1, 2, 3), unit=u.deg)  # (d, m, s)
-      Angle(10.2345 * u.deg)
-      Angle(Angle(10.2345 * u.deg))
+      >>> Angle('10.2345d')
```

Comment:
these doctested examples now need to be full examples, with appropriate imports, etc.

Reply:
do you mean that the first shown example needs to be preceded by the imports required to declare an instance of this class?

Reply:
Yes. If you copy paste these  as-is into an interpreter it should be able to run without issues.